src/components/LoginForm.js

```diff
@@ -0,0 +1,29 @@
+const LoginForm = () => {
+  return (
+    <form className="login-form">
+      <h3 className="login-form__head mobile--hidden "> Login </h3>
```

Comment:
```suggestion
      <h3 className="login-form__head mobile--hidden"> Login </h3>
```

Please remove this space 🥰